contrib/export/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/tendermint/tendermint/types"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"v0.36": v036.Migrate,
+	}
+	source        string
+	target        string
+	importGenesis string
+)
+
+func init() {
+	log.SetPrefix("")
+	log.SetFlags(0)
+
+	// this flag seems unnecessary, we can reintriduce it once we support multiple versions migration at once
+	flag.StringVar(&source, "source", "v0.34", "The SDK version that exported the genesis")
+	flag.StringVar(&target, "target", "", "The target SDK genesis version")
```

Comment:
This too should be a positional argument

Reply:
so the command will be:
`go run ../export/main.go v0.36 ~/.gaiad/config/genesis.json > ~/genesis.new.json`
`go run ../export/main.go v0.36 ~/.gaiad/config/genesis.json --source v0.33 > ~/genesis.new.json`